packages/kit/src/views/AccountManagerStacks/pages/AccountSelectorStack/WalletDetails/AccountValue.tsx

```
@@ -99,38 +92,23 @@ function AccountValueWithSpotlight({
-  const shouldShowSpotlight = isFocused && !isOthersUniversal && index === 0;
-  const intl = useIntl();
-  return (
-    <Spotlight
-      delayMs={300}
-      containerProps={{ flexShrink: 1 }}
-      isVisible={shouldShowSpotlight}
-      message={intl.formatMessage({
-        id: ETranslations.spotlight_enable_account_asset_message,
-      })}
-      tourName={ESpotlightTour.allNetworkAccountValue}
+  return accountValue && accountValue.currency ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)